cfme/networks/views.py

```diff
@@ -420,7 +420,7 @@ class NetworkRouterAddView(BaseLoggedInPage):
     """ Represents Add NetworkRouters page """
     network_manager = Select(id='ems_id')
     router_name = TextInput(name='name')
-    ext_gateway = BootstrapSwitch(id='network_router_external_gateway')
+    ext_gateway = BootstrapSwitch(name='external_gateway')
```

Comment:
Question : Is this both for 5.9 and 5.10 ?

Reply:
Yes the same for 5.8, 5.9 and 5.10